plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -507,18 +507,11 @@ instance (Closed uni, GShow uni, uni `Everywhere` PrettyConst, Pretty fun) =>
 type instance UniOf (CekValue uni fun) = uni
 
 instance FromConstant (CekValue uni fun) where
-    fromConstant val = VCon val
+    fromConstant = VCon
```

Comment:
That's just faster, given that `fromConstant` should not get inlined and should be passed as a dictionary at runtime.